demos/text_to_speech_demo/python/README.md

```diff
@@ -63,7 +63,7 @@ Options:
                         Required. Path to ForwardTacotron`s mel-spectrogram
                         regression part (*.xml format).
   -i INPUT, --input INPUT
-                        Required. Text file with text.
+                        Optional. Text file with text. Replaces console input.
```

Comment:
Optional means you can ommit `-i` at all, what seems not the case. You probably want to say something like: Path to the text file or just string of text to translate to speech.

Reply:
Current implementation allows to pass the input in 3 ways: txt file, sentences in format "text1" "text2"..., and the `-i` can be omitted, in this case the user might write the text at the demo runtime (console input)
I think it's better to extend the description like this:
`Optional. Path to the .txt file or string of text to translate to speech. Also, supports the console input.`

Reply:
not sure it is good idea, all demo use `-i` to specify input and this still not interactive, you can only enter sentence once, which provides no difference from from the case where you enter sentence as param of `-i` option, and this demo do not need interactivity as it can output wav files only.

when I try to run demo without `-i` with text-to-speech-en-multi-0001 models and enter hello world sentence and press enter I get error:
RuntimeError: Can't SetBlob with name: speaker_embedding, because model input and blob are incompatible
besides error itself, please replace SetBlob with OV2.0 method name

Reply:
@VoronovaIntern could you please remove the `sentences()` function (since it provides the capability to write input in a console mode) and just iterate through lines in `input_data`?

Reply:
@vladimir-dudnik  Yes